libs/utils/utils.js

```diff
@@ -176,6 +176,27 @@ export function getMetadata(name, doc = document) {
   return meta && meta.content;
 }
 
+const handleEntitlements = (() => {
+  let entResolve;
+  const entPromise = new Promise((resolve) => {
+    entResolve = resolve;
+  });
+
+  return (resolveVal) => {
+    if (resolveVal !== undefined) {
+      entResolve(resolveVal);
+    }
+    return entPromise;
+  };
+})();
+
+function setupMiloObj(config) {
+  window.milo ||= {};
```

Comment:
one issue I noticed on https://cpeyer-aep--milo--adobecom.hlx.page/ - `window.milo` will return an element with ID 'milo' instead of an Object with `deferredPromise`

Reply:
Ugh - looks like if you have a title or anchor named `#milo` then `window.milo` will point to that.  This is an existing issue.  Since entitlements is now not using the window.milo object, and `window.milo.deferredPromise` still exists and works I think we can leave as is.